PORT-11785 Github App installation guide refactoring (#1820)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/git/github/github.md

# GitHub

Port's GitHub integration allows you to model GitHub resources in your software catalog and ingest data into them.

## Overview

This integration allows you to:

- Map and organize your desired GitHub resources and their metadata in Port (see supported resources below).
- Watch for GitHub object changes (create/update/delete) in real-time, and automatically apply the changes to your software catalog.
- Manage Port entities using GitOps.
- Trigger GitHub workflows directly from Port.

### Supported resources

The resources that can be ingested from GitHub into Port are listed below.
It is possible to reference any field that appears in the API responses linked below in the mapping configuration.

 <GitHubResources/>

## Setup

To install Port's GitHub app, follow these steps:

1. Go to the [GitHub App page](https://github.com/apps/getport-io).

> 2. Choose the organization you want the app to be installed for.

Reviewer: instead of 2 steps, 2&3 should be one step:
`Choose the organization or repositories in which you want the app to be installed`.

Also, add a "tip" admonition that says that we recommend to install in the entire org to ensure that all data is ingested.

Also, are you sure that this is the order of the steps? You choose organization/repos first, and then click install? I thought it was the other way around, please check.

Author: Yeah I am certain of the order. I implemented it first. When you click on Install, and the github account is in multiple organizations, they will ask you to select the org first. and then after that choose a repo or allow all repos access. 

The admonition is also a bit confusing. do you mean we recommend to allow all repository to ensure all data is ingested ?

3. Choose the repositories you want the app to be installed for. 

4. Click on the `Install` button.

> 5. You'll be redirected to Port, please **log in**.

Reviewer: im not sure this step is needed

Author: okay

# GitHub

Port's GitHub integration allows you to model GitHub resources in your software catalog and ingest data into them.

## Overview

This integration allows you to:

- Map and organize your desired GitHub resources and their metadata in Port (see supported resources below).
- Watch for GitHub object changes (create/update/delete) in real-time, and automatically apply the changes to your software catalog.
- Manage Port entities using GitOps.
- Trigger GitHub workflows directly from Port.

### Supported resources

The resources that can be ingested from GitHub into Port are listed below.
It is possible to reference any field that appears in the API responses linked below in the mapping configuration.

 <GitHubResources/>

## Setup

To install Port's GitHub app, follow these steps:

1. Go to the [GitHub App page](https://github.com/apps/getport-io).

2. Click on the `Configure` button.

> 3. Choose the organization you want the app to be installed for.

Reviewer: ```suggestion
3. Choose the organization in which to install the app.
```

> 4. Choose the repositories you want the app to be installed for. 

Reviewer: ```suggestion
4. Within the selected organization, choose the repositories in which to install the app.
```

>     :::tip Installing for all repositories
>      We recommend to install for all repositories to ensure that all data is ingested.
>     :::

Reviewer: ```suggestion
```
